### ISSUE TITLE
iOS 9.3 allow button not works

### ISSUE THREAD

**Issue by howardsun-tw** [nickoneill/PermissionScope#213]
![image](https://cloud.githubusercontent.com/assets/17197816/21738845/c0fbe29a-d4c7-11e6-815f-b2144c0caa5c.png)
iOS 10.0 all works fine.
However, on iOS 9 Device, when i tapped allowButton , then it just turned into denied.
Is there anyone knows what 's going on?
I will be grateful for any help you can provide.

**Comment by bre7** [comment]
Please test using a real device 👍

**Comment by rajubd49** [comment]
I'm also having problems in iOS 10.2 simulator. Contacts permission is denied automatically If I press enable button. Location permission enable button tap is not working.

**Comment by nickoneill** [comment]
If it wasn't clear before @rajubd49, please test on a real device. The simulator is not a reliable place for permissions testing.

**Comment by rajubd49** [comment]
Yes, it works on real device. But we don't always get the chance to test on real device. That's why I raised this issue. Anyway, thanks for your reply.
